src/client/src/apps/MainRoute/widgets/footer-version/FooterVersion.tsx

```diff
@@ -1,12 +1,51 @@
-import React, { FC } from 'react'
+import React, { FC, useState } from 'react'
 import styled from 'styled-components/macro'
 
 export const Wrapper = styled.div`
   color: ${props => props.theme.textColor};
   opacity: 0.59;
   font-size: 0.75rem;
 `
+export const Link = styled.a`
+  color: inherit;
+  text-decoration: inherit;
+`
+const FooterVersion: FC = () => {
+  const [versionExists, setVersionExists] = useState<boolean | void>(false)
+  const gitTagExists = async (gitTag: string | undefined) => {
+    const exists = await fetch(
+      'https://api.github.com/repos/AdaptiveConsulting/ReactiveTraderCloud/releases'
+    )
+      .then(response => response.json())
+      .then(data => {
+        for (let i = 0; i < data.length; i++) {
+          if (data[i].tag_name === gitTag) {
```

Comment:
This is not wrong, but there are more readable ways to filter arrays in JS. E.g. `Array.prototype.find`.

Also DON'T mix async-await keywords with then and catch promise callbacks. Use one or the other.

Reply:
Oh I see. If I don't use "then" will I be able to simply return a boolean from my async function rather than a promise?